segmentation_labeling_app/rois/rois.py

```diff
@@ -133,31 +137,28 @@ def roi_from_query(cls, roi_id: int) -> "ROI":
 
         Returns: ROI object for the given segmentation_id and roi_id
         """
-        label_vars = query_utils.get_labeling_env_vars()
 
-        roi = query_utils.query(
-            f"SELECT * FROM rois WHERE id={roi_id}",
-            user=label_vars.user,
-            host=label_vars.host,
-            database=label_vars.database,
-            port=label_vars.port,
-            password=label_vars.password)[0]
+        roi = db_conn.query(f"SELECT * FROM rois WHERE id={roi_id}")[0]
 
-        segmentation_run = query_utils.query(
+        segmentation_run = db_conn.query(
             ("SELECT * FROM segmentation_runs WHERE "
-             f"id={roi['segmentation_run_id']}"),
-            user=label_vars.user,
-            host=label_vars.host,
-            database=label_vars.database,
-            port=label_vars.port,
-            password=label_vars.password)[0]
+             f"id={roi['segmentation_run_id']}"))[0]
+
+        # NOTE: temporary legacy support for no traces
```

Comment:
Can you please add `# TODO` to this? Using a standard makes it easy to search for later.

Reply:
changed NOTEs to TODOs

Reply:
removed support for traceless ROIs and deleted this TODO